cmdline.h

```diff
@@ -61,7 +61,7 @@ std::vector<std::string_view> expand_response_files(C &ctx, char **argv) {
   return vec;
 }
 
-static std::string_view string_trim(std::string_view str) {
+[[maybe_unused]] static std::string_view string_trim(std::string_view str) {
```

Comment:
Instead of adding an attribute, can you change this to `static inline`?

Reply:
Done.